iohk/UpdateLogic.hs

```diff
@@ -349,8 +353,8 @@ makeDaedalusVersion ver = makeDaedalusVersion' ver . T.pack . show
 makeDaedalusVersion' :: T.Text -> T.Text -> ApplicationVersion a
 makeDaedalusVersion' ver num = ApplicationVersion $ ver <> "." <> num
 
-processDarwinBuildTravis :: T.Text -> T.Text -> T.Text -> BuildId -> (ApplicationVersionKey 'Win64, ApplicationVersionKey 'Mac64) -> T.Text -> IO (GlobalResults, CiResult)
-processDarwinBuildTravis daedalus_rev daedalus_version tempdir buildId versionKey ciUrl = do
+processDarwinBuildTravis :: BK.APIToken -> T.Text -> T.Text -> T.Text -> BuildId -> (ApplicationVersionKey 'Win64, ApplicationVersionKey 'Mac64) -> T.Text -> IO (GlobalResults, CiResult)
```

Comment:
Buildkite token here is just to match `processDarwinBuildKite` signature? Maybe we can turn this into a record :)

Reply:
Yes a `Config` record type might be useful and increase clarity.
However this travis code will be deleted imminently in #256 .

Reply:
Good point, let's leave it for now :)

Reply:
Thanks, merging ...